homeassistant/components/persistent_notification/__init__.py

```diff
@@ -113,7 +113,9 @@ def create_service(call):
             _LOGGER.error('Error rendering message %s: %s', message, ex)
             message = message.template
 
-        hass.states.async_set(entity_id, message, attr)
+        attr[ATTR_MESSAGE] = message
+
+        hass.states.async_set(entity_id, None, attr)
```

Comment:
A entity should have a state...

Reply:
Yes, should, but in this case, it is a question what should be there. Notification ID ? Because Title and Message are not “safe” because of length

Reply:
Add `notifying`. We do a similar thing for zones which just have state `zoning`.

Reply:
State changed to `notifying`